lib/models/introductory_price.g.dart

```diff
@@ -11,10 +11,10 @@ _$IntroductoryPriceImpl _$$IntroductoryPriceImplFromJson(Map json) =>
       (json['price'] as num).toDouble(),
       json['priceString'] as String,
       json['period'] as String,
-      json['cycles'] as int,
+      (json['cycles'] as num).toInt(),
       $enumDecode(_$PeriodUnitEnumMap, json['periodUnit'],
           unknownValue: PeriodUnit.unknown),
-      json['periodNumberOfUnits'] as int,
+      (json['periodNumberOfUnits'] as num).toInt(),
```

Comment:
New freezed version?

Reply:
yes, the build was failing because CI was using the newer version of freezed so I had to rerun the build of files. I could split it into another PR, but I think it's fine since they are autogenerated?

Reply:
Yeah I think it's fine